### DISCUSSION TITLE
Cannot load ogg files for transcribing

### DISCUSSION THREAD

**Discussion by zaperda** [open-webui/open-webui#4240] [General]
# Bug Report

## Description
There is a recording of a telegram conversation in ogg format . I am trying to transcribe the recording into text format. I can't upload the file, I get an error Unknown file type audi/ogg. 
The ogg format is native to whisper, it understands it.

![image](https://github.com/user-attachments/assets/75b9a355-52e2-460c-ac63-be8da4ed6725)

## Environment

- **Open WebUI Version:** [0.3.10]
- **Ollama (if applicable):** [0.3.0]

- **Operating System:** [Windows 11, MacOS]
- **Browser (if applicable):** [Chrome, Safari]

**Comment by tjbck** [comment]
Sample ogg file would be extremely helpful here!

**Reply by zaperda** [reply]
> Sample ogg file would be extremely helpful here!

https://upload.wikimedia.org/wikipedia/commons/1/1f/George_W_Bush_Columbia_FINAL.ogg

**Comment by zaperda** [comment]
In version 0.3.12 ogg files are not loaded, while whisper itself transcribes these formats

**Comment by hola-ivan** [comment]
I am experiencing the same issue - thanks in advance for any guidance!

**Comment by tjbck** [comment]
Added to dev!
